### DISCUSSION TITLE
How to apply for pro badge

### DISCUSSION THREAD

**Discussion by bronglil** [community/community#63791] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

I have finished my bachelor's degree and am now applying for the github PRO badge. However, I'm facing an issue because the university's email address is not available to us during our studies. Instead, we used our personal email addresses. Unfortunately, I'm unable to apply using my personal email address.

**Comment by little-software-engineer** [comment, marked as answer]
Hello, here are a few methods you can try to solve your problem:

Contact GitHub Support: Reach out to GitHub's support team and explain your situation. They might be able to provide guidance or make an exception if you can provide alternative documentation to prove your academic eligibility.

Use University Documents: If you have any official documents from your university, such as a diploma, transcript, or certificate, you could try submitting those along with your personal email address to verify your eligibility.

Check with University IT: Contact your university's IT department or registrar's office to see if there's a way to access or activate your university email address even after graduation. Some universities may offer extended access to email services for alumni.

LinkedIn or Social Media: Some verification processes allow you to verify your academic status through social media profiles, such as LinkedIn. If you have your education information listed on LinkedIn, you could try connecting your profile to verify your student status.

A friend of mine had a similar problem a couple of months ago, so he used method number 2 and submitted a certificate from his university that proved he was a student (you should check with your university to see if they can provide a similar document). Best of luck with your application!

**Reply by bronglil** [reply]
where he had submitted the document?  through support email.

**Reply by little-software-engineer** [reply]
I think the GitHub Pro membership setup for students asks you whether you have such a document, and then you are also able to upload it there. But I think you can also try sending these documents, once acquired from the university, to the support mail.

**Comment by melvincwng** [comment]
@bronglil, in order to get the GitHub Pro badge, you need to subscribe to the GitHub Pro plan. 

See which options below best apply to your situation:
- If you are still a **student** with a valid school email address, apply for the [GitHub Student Developer pack](https://education.github.com/pack) - which provides you free access to GitHub Pro (while you remain a student).
- If you are **no longer a student**, you can subscribe to GitHub Pro on your own which should cost $4 USD monthly.

For steps on how to subscribe to GitHub Pro on your own, you can refer to: [GitHub Documentation - Step by Step instructions on how to subscribe to GitHub Pro plan](https://docs.github.com/en/billing/managing-the-plan-for-your-github-account/upgrading-your-accounts-plan#upgrading-your-personal-accounts-plan)

Additional reference: [Benefits of GitHub Pro plan](https://docs.github.com/en/get-started/learning-about-github/githubs-plans#github-pro)

Hope this answers & resolves your question, take care and have a good day ahead! :smile:

**Reply by bronglil** [reply]
I'll think about it if there no any other way.

**Comment by npneeraj** [comment]
Step 2. Check with University IT: Contact your university's IT department or registrar's office to see if there's a way to access or activate your university email address even after graduation. Some universities may offer extended access to email services for alumni.
It didn't worked on my case.
